index.js

```diff
@@ -83,7 +83,7 @@ function uglifyify(file, opts) {
 
       map.setProperty('sources', [path.basename(file)])
       map.setProperty('sourcesContent', matched
-        ? opts.inSourceMap.sourcesContent
+        ? opts.sourceMap.sourcesContent
```

Comment:
oh! you're right, this should be `sourceMap`, and I think it should even be `sourceMap.content` like on [line 69](https://github.com/hughsk/uglifyify/pull/67/files#diff-168726dbe96b3ce427e7fedce31bb0bcR69)